pkg/remote/aws/repository/appautoscaling_repository.go

```diff
@@ -0,0 +1,87 @@
+package repository
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/applicationautoscaling"
+	"github.com/aws/aws-sdk-go/service/applicationautoscaling/applicationautoscalingiface"
+	"github.com/cloudskiff/driftctl/pkg/remote/cache"
+)
+
+type AppAutoScalingRepository interface {
+	ServiceNamespaceValues() []string
+	DescribeScalableTargets(string) ([]*applicationautoscaling.ScalableTarget, error)
+	DescribeScalingPolicies(string) ([]*applicationautoscaling.ScalingPolicy, error)
+	DescribeScheduledActions(string) ([]*applicationautoscaling.ScheduledAction, error)
```

Comment:
Are theses methods useful ? I do not see any usage within enumerator.

Reply:
They will be used by #1028  & #1027, as these 3 resources must be implemented together

Do you want me to split accordingly ?

Reply:
Yeah it's better to keep PR atomic 🙏🏻